charts/ccd-data-store-api/values.yaml

```diff
@@ -89,3 +89,7 @@ java:
     DATA_STORE_CROSS_JURISDICTIONAL_ROLES: caseworker-caa,caseworker-approver
     DATA_STORE_CITIZEN_ROLES: citizen,letter-holder
     DATA_STORE_SECURITY_LOGGING_FILTER_PATH_REGEX: '.*'
+
+    AM_ROLE_ASSIGNMENT_PAGINATION_ENABLED: false
```

Comment:
Better setting the default values in app properties only, to avoid chart change.
Otherwise needs chart version bump up, or master pipeline will fail.

Reply:
Removed